pyiron_contrib/atomistics/runner/job.py

```diff
@@ -293,10 +225,34 @@ def publication(self):
             ]
         }
 
+    @property
+    def scaling(self):
+        return self.storage.scaling
+
+    @property
+    def weights(self):
+        return self.storage.weights
+
+    @property
+    def sfvalues(self):
+        return self.storage.sfvalues
+
+    @property
+    def splittraintest(self):
+        return self.storage.splittraintest
+
+    @property
+    def input(self):
+        return self.storage.input
+
+    @property
+    def output(self):
+        return self.storage.output
+
     @property
     def structures(self):
         """Store a dataset consisting of many chemical structures."""
-        return self._structures
+        return self.storage.structures
 
     @structures.setter
     def structures(self, structures):
```

Comment:
Should document that this appends structures, since this will not be visible in code, i.e.
```python
job.structures = [structure1, structure2]
job.structures = [structure3, structure4]
```
(where `structures` are `Atoms`) 
will look like it only contains 3 & 4, but actually contains all of them.